snapshot-labs/snapshot-sidekick
# fix: fix fetchVotes progression not incrementing

## 🧿 Current issues / What's wrong ?

While fetching the votesReport, the % progression is never incremented, and stays at 0 until the end.

This is probably due to a merge conflict resolution gone bad.

## 💊 Fixes / Solution

- Increment the progress while fetching the votes

## 🚧 Changes

- Add back the progress incrementation, while paginating the votes fetching request

## 🛠️ Tests

- Run `yarn dev`
- Run `curl -X POST localhost:3005/api/votes/0xda4f201a37ea08cf1892418e7b9e88f5687a68dbdc96c3ab22abaa1c7244648e`
- Observe the console, the progress % should increment (slowly, this proposal has 2M votes), may takes 30s
